### ISSUE TITLE
Suggestion: possibility to mark a praise when it lacks clarity/information

### ISSUE THREAD

**Issue by PhilH** [CommonsBuild/praise_frontend#144]
Right now there's an option to dismiss, that I used when the praise shouldn't be taken into account as per the "Rules of Praise and Quantification" - for instance when it's about tweeting or attending a meeting.

It would be cool to be able to flag praises that are obscure. I dismissed the ones I couldn't understand, or scored them in a way that is more approximate than others. I'd like to be able to flag so that praisers get the feedback and understands what needs to be more precise in their future praises.

I can give examples :)

**Comment by kristoferlund** [comment]
Yes! We would def like to implement something like this in a future version. Not only notifying praiser that the contents is "obscure" but also asking for clarification.
